sdk/messaging/azservicebus/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.4.1 (Unreleased)
+## 1.0.0 (2022-05-12)
```

Comment:
You should release a `v1.0.0` of `messaging/internal` first.

Reply:
Updating with offline conversation - we're going to do a few things to jettison `internal`, but this PR will get us so we can release a new preview release of azservicebus with the latest azcore. I'll do a release of `messaging/internal` and then push another PR to bump the internal reference in here.